### PR TITLE
second return of fastmos: electric boogaloo

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

re-enables the ''''''''fixed''''''' auxmos monstermos

## Why It's Good For The Game

the people demand it, i love democracy

## Changelog
:cl:
add: atmos made fast once again
fix: fixes monstermos
/:cl:

### CODE REVIEW

**Comment by ZombiesVsPlants** [comment]
image

**Comment by williamcll** [comment]
What did you fix exactly?

**Comment by Sinestia** [comment]
> 
> 
> What did you fix exactly?

Most of the magic is elsewhere, they've been working with putnam on this.

**Comment by ToasterBan** [comment]
does this have the monstermos flaw of only 14 gas "slots"?

**Comment by MNarath1** [comment]
Could you maybe PR the auxmos changes to the auxmos fork ? Pretty sure those two sides should get PR on those two side respectivly instead of just the compiled dll here.

**Comment by MNarath1** [comment]
ah wait you did on putnam's repro upstream

**Comment by Sinestia** [comment]
> 
> 
> does this have the monstermos flaw of only 14 gas "slots"?

that wasn't a flaw, it's a single preprocessor line that needed to be changed.

Extools just has to know how many gasses to expect, you can set the number to whatever you want. I think ike just left it at 14 and threw in a dummy gas so that he could avoid needing to make any direct modifications of extools. 

(auxmos does do it dynamically however, but fastmos won't change that. So the answer to that question is no)

**Comment by jupyterkat** [comment]
> does this have the monstermos flaw of only 14 gas "slots"?

no

**Comment by PowerfulBacon** [comment]
time to remap all station firelocks again :tf:

**Comment by PowerfulBacon** [comment]
imo LINDA is too slow and makes breaches and space threats nothing, Monstermos is too fast and makes minor breaches absolutely devestating and makes the game need the hugbox firelocks.

**Comment by AnCopper** [comment]
> imo LINDA is too slow and makes breaches and space threats nothing, Monstermos is too fast and makes minor breaches absolutely devestating and makes the game need the hugbox firelocks.

Paradise Station has fastmos that doesn't instantly depressurize the entire Station.

**Comment by ike709** [comment]
> Paradise Station has fastmos that doesn't instantly depressurize the entire Station.

Get someone to implement ZAS in auxtools and I'll consider it.

**Comment by Sinestia** [comment]
> 
> 
> > Paradise Station has fastmos that doesn't instantly depressurize the entire Station.
> 
> Get someone to implement ZAS in auxtools and I'll consider it.

paradise has LINDA

tbh normal auxmos will be improved a lot when it's tweaked and the bugs are fixed

especially when some of our old nerfs to LINDA are reverted, as they aren't needed anymore.

**Comment by jupyterkat** [comment]
up for testmerge

**Comment by PowerfulBacon** [comment]
Can we put this to vote again now thar people have had time to play without monstermos? I was originally pro monstermos, however going back to old atmos I much prefer the non instant decompression.

**Comment by jupyterkat** [comment]
> Can we put this to vote again now thar people have had time to play without monstermos? I was originally pro monstermos, however going back to old atmos I much prefer the non instant decompression.

no more votes, the fate is sealed

**Comment by Emiliapains** [comment]
In any case I vote for this to merge, uh wish we could add a poll in here

**Comment by mcmeiler** [comment]
> add a poll in here

We have a PR vote channel in the discord for a reason

**Comment by francinum** [comment]
I'm not a fan of bringing back the quick escape firelocks.

File: code/game/machinery/doors/firedoor.dm [hunk 1]
```diff
@@ -73,10 +73,19 @@
 	return ..()
 
 /obj/machinery/door/firedoor/Bumped(atom/movable/AM)
-	if(panel_open || operating)
```
Comment (francinum) [inline code/game/machinery/doors/firedoor.dm:76]: Please revert your changes to this file.

**Review by francinum — CHANGES_REQUESTED** [review]

**Comment by jupyterkat** [comment]
all works on this will be halted until francis learnt his lesson

**Comment by Emiliapains** [comment]
Bring it back :(

**Pull request closed without merging**